modules/evm/rpc/src/lib.rs

```diff
@@ -387,22 +404,21 @@ where
 		// rpc_binary_search_estimate block
 		{
 			// Define the lower bound of the binary search
-			const MIN_GAS_PER_TX: U256 = U256([21_000, 0, 0, 0]);
-			let mut lowest = MIN_GAS_PER_TX;
+			let mut lowest = BASE_CALL_GAS;
```

Comment:
<img width="343" alt="image" src="https://user-images.githubusercontent.com/16496415/165208212-b12e7eae-1ca2-4fd4-8ed9-fda820812816.png">
The minimum value should not be changed. This affects the lower limit of gas_limit

Reply:
@zjb0807 we can leave that 21k, no problem